htmlandjslab.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title> 
+          Bio for a Stranger </title>
+          <script src="./index2.js">
+            //   alert("You are about to enter a dangerous site")
+            //   let userAge = prompt("Enter your age to continue")
+            //   console.log(Number(userAge) + 50)
+
+
+          </script>
+
+
+
+
+
+
+    </head>
+    <body>
+           <a href="contactme.html">Contact Me </a>
+           <hr>
+
+
+
+
+<h1 id = "top">A Person</h1>
+<br>Hello, I'm a stranger. Try to find out about me. 
+
+</p>
+<h2 id = "num">I want to be a number way more than I want to be a string </h2>
+</hr>
+<h2>Current list of My Passions as of November 2019</h2>
+
+<h1 id="top">Name</h1>
+       </h1>
```

Comment:
Properly indent your html much like you would code, this would make it much easier to read.